templates/support/index.html

```diff
@@ -640,12 +750,47 @@ <h3 class="p-heading--5">Modular MLOps platforms</h3>
           <h3 class="p-heading--5">Cost-effective private and hybrid cloud</h3>
         </div>
         <div class="col-6 col-medium-3">
-          <p>Rely on Canonical's <a href="/openstack">Charmed OpenStack</a> to build a cost-effective hybrid multi-cloud infrastructure with the peace of mind of experts on your side.</p>
-          <p>Want to deploy your clouds in minutes, with just a simple command? Check out Canonical's <a href="https://canonical.com/microcloud">Microcloud</a>, the low-touch cloud solution, optimized for easy and repeatable deployments.</p>
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-wrapper">
+                {{ image (
+                  url="https://assets.ubuntu.com/v1/6ac07c33-canonical-openstack.png",
+                  alt="",
+                  width="126",
+                  height="34",
+                  hi_def=True,
+                  loading="lazy"
+                  ) | safe
+                }}
+              </div>
+            </div>
+            <div class="col-3">
+              <p>Rely on <a href="/openstack">Canonical OpenStack</a> to build a cost-effective hybrid multi-cloud infrastructure with the peace of mind of experts on your side.</p>
+            </div>
+          </div>
+          <hr class="p-rule">
+          <div class="row">
+            <div class="col-3">
+              <div class="p-image-wrapper">
+                {{ image (
+                  url="https://assets.ubuntu.com/v1/eab36658-MicroCloud%20logo@2x.png",
+                  alt="",
```

Comment:
I believe @juanruitina will ask us to fill this up.

Reply:
Yes! Can you please link both logos and add the alt text? People looove to click on images.